src/lint/engine/ArcanistLintEngine.php

```diff
@@ -477,86 +477,82 @@ private function executeLinters(array $runnable) {
     assert_instances_of($runnable, 'ArcanistLinter');
 
     $all_paths = $this->getPaths();
-    $path_chunks = array_chunk($all_paths, 32, $preserve_keys = true);
 
     $exception_lists = array();
-    foreach ($path_chunks as $chunk) {
-      $exception_lists[] = $this->executeLintersOnChunk($runnable, $chunk);
+    foreach ($runnable as $linter) {
+      $chunk_size = $linter->getChunkSizeLimit() ? $linter->getChunkSizeLimit() : count($all_paths);
```

Comment:
What would be false here, 0?

Reply:
This evaluate null as well, but both works